src/index.js

```diff
@@ -22,8 +24,35 @@ var _ = require('lodash'),
 
 var BROWSER_SCIPTS_DIR = path.join(__dirname, 'browser-scripts');
 
+var levels = [
+  'error',
+  'warn',
+  'info',
+  'debug',
+];
+var instanceNumber = 0;
 
+function bindLogger(logLevel, kind) {
+  var logger = glogg('gulp-server-livereload-' + kind + '-' + instanceNumber);
 
+  logLevel = levels.indexOf(logLevel) + 1;
+
+  if (logLevel == 0) {
+    throw 'Logging level "' + logLevel + '" does not exist!';
+  }
+
+  levels
+    .filter(function (item, i) {
+      return i < logLevel;
+    })
+    .forEach(function (level) {
+      logger.on(level, function () {
+        gulpLogger.info.apply(gulpLogger, arguments);
```

Comment:
The intention here is to ensure that the logging makes it through our context and then `gulp-cli`'s context without being additionally filtered - this simplifies configuration as you'd have to run `gulp-cli` with the most verbose switch to see debug messages from livereload or the client.

This way, the logs are collapsed down so that the config can be changed in one place, and you're either guaranteed not to see anything (if running gulp in a less verbose mode) or, see everything _you want_ to see.